examples/tv-casting-app/APIs.md

```diff
@@ -819,13 +833,14 @@ response types to use with these APIs:
 
 ### Issuing Commands
 
-_{Complete Command invocation examples: [Linux](linux/simple-app-helper.cpp)}_
+_{Complete Command invocation examples: [Linux](linux/simple-app-helper.cpp) |
+[iOS](darwin/TvCasting/TvCasting/MCContentLauncherLaunchURLExampleViewModel.swift)}_
 
-The Casting Client can get a reference to a `endpoint` on a `CastingPlayer`,
+The Casting Client can get a reference to an `Endpoint` on a `CastingPlayer`,
```

Comment:
Should this be an MCEndpoint?

``````suggestion
The Casting Client can get a reference to an `MCEndpoint` on a `CastingPlayer`,
```

Reply:
The prefix "MC" is only required and used for the iOS version of these APIs, not Linux/Android. We have used the "MC" prefix when describing the iOS version further down below.